frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -38,7 +45,7 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({ error, loaded, refresh, o
         latest {LOG_TAIL_LINES} lines. Exceptionally long lines are abridged. To view the full log
         for this task, you can{' '}
         <Button
-          isDisabled={!onDownload}
+          isDisabled={!onDownload || isLogsNotAvailable}
           variant="link"
           isInline
           component="span"
```

Comment:
I don't see why this `Button` uses `component="span"`. An inline button should work.